staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -402,6 +404,16 @@ func DefaultOpenAPIConfig(getDefinitions openapicommon.GetOpenAPIDefinitions, de
 	}
 }
 
+func DefaultOpenAPIV3Config(getDefinitions openapicommon.GetOpenAPIDefinitions, defNamer *apiopenapi.DefinitionNamer) *openapicommon.Config {
```

Comment:
Can you add a godoc comment or make the name more informative? From reading this function I don't know what it does. Maybe I'll figure it out as I read more.